app/src/main/res/values/strings.xml

```diff
@@ -149,7 +149,7 @@
     <string name="check_balance_title">Check balance</string>
     <string name="check_balance_desc">Nice! You added your first account. Now you can check your balance on Stax.</string>
     <string name="check_balance_alt">Nice! Account added successfully. Now you can check your balance on Stax.</string>
-    <string name="check_balance_alt_plural">Nice! Accounts added successfully. Now you can check your balance on Stax.</string>
+    <string name="check_balance_alt_plural">Nice! Accounts added successfully. Now you can check your balances on Stax.</string>
```

Comment:
I wonder whether it will make sense to say `You can now check your balances on Stax`